### PR TITLE
A script to fix the tiffs

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?

Nothing (unless you run it, then it changes the image it's fed)

## How can success be measured?

Our poor tiffs that are now pngs have their metadata saved outside elasticsearch, and will reingest.

Run `./run.sh BUCKET $ID` with a grid-cli profile set up it'll download an image from the grid, clear the meta and put back the original meta, and the filemeta in a special xmp field. This can be inspected using exiftool `exiftool -xmp -b $ID | xq` (where xq is from `brew install python-yq`) to get fileMetadata run ` exiftool -xmp -b $ID  | xq '.["x:xmpmeta"]' | jq '.["rdf:RDF"]' | jq '.["rdf:Description"][1]' | jq '.["gridImage:FileMetadata"]' | jq fromjson` 
To see the rest, upload the modified image to grid TEST/local and examine.

We will need to test reingestion of these images before running against the bulk.

depends on https://github.com/guardian/grid/pull/3249
## Tested?
- [x] locally by committer
- [ ] locally by Guardian reviewer
- [ ] on the Guardian's TEST environment

### CODE REVIEW

File: scripts/inadvisable-metadata-backup/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    // "types": [],                                 /* Type declaration files to be included in compilation. */
+    // "allowSyntheticDefaultImports": true,        /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
+    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
+    // "preserveSymlinks": true,                    /* Do not resolve the real path of symlinks. */
+    // "allowUmdGlobalAccess": true,                /* Allow accessing UMD globals from modules. */
+
+    /* Source Map Options */
+    // "sourceRoot": "",                            /* Specify the location where debugger should locate TypeScript files instead of source locations. */
+    // "mapRoot": "",                               /* Specify the location where debugger should locate map files instead of generated locations. */
+    // "inlineSourceMap": true,                     /* Emit a single file with source maps instead of having a separate file. */
```
Comment (jonathonherbert) [inline scripts/inadvisable-metadata-backup/tsconfig.json:60]: We don't need this lot!

File: scripts/inadvisable-metadata-backup/run.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+ID=$2
+BUCKET=$1
+
+echo "Processing $ID in $BUCKET"
+
+grid image:download -d=. $ID
+grid image:get --hyrdate $ID > $ID.json
```
Comment (jonathonherbert) [inline scripts/inadvisable-metadata-backup/run.sh:9]:
```suggestion
grid image:get --hydrate $ID > $ID.json
```

**Comment by jonathonherbert** [comment]
Code looks fine, happy to give this a test but suspect @paperboyo will be front of the queue 😁 – be delighted to join in!

Probably best to wait 'til we know we're happy w/ the result before approving.

File: scripts/inadvisable-metadata-backup/run.sh (hunk 2 above)
Comment (AWare) [reply scripts/inadvisable-metadata-backup/run.sh:9]: whoooops

**Comment by AWare** [comment]
missing the following: 

- [ ]  credit
- [ ]  byline
- [ ]  copyright
- [ ]  specialInstructions
- [ ]  keywords

File: scripts/inadvisable-metadata-backup/run.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+BUCKET=$1
+
+echo "Processing $ID in $BUCKET"
+
+grid image:download -d=. $ID
+grid image:get --hydrate $ID > $ID.json
+sips --setProperty format png $ID --out $ID
+exiftool -all= $ID
+yarn update $ID
+# yarn s3 $ID $BUCKET
```
Comment (aug24) [inline scripts/inadvisable-metadata-backup/run.sh:14]: Should this line be uncommented?!

**Review by aug24 — APPROVED** [review]

File: scripts/inadvisable-metadata-backup/run.sh (hunk 3 above)
Comment (AWare) [reply scripts/inadvisable-metadata-backup/run.sh:14]: Yeah.

**Review by aug24 — APPROVED** [review]

**Pull request merged**